common/libs/nodes/rpc/index.ts

```diff
@@ -24,78 +26,73 @@ export default class RpcNode implements INode {
   }
 
   public sendCallRequest(txObj: TxObj): Promise<string> {
-    return this.client.call(this.requests.ethCall(txObj)).then(r => {
-      if (r.error) {
-        throw Error(r.error.message);
-      }
-      return r.result;
-    });
+    return this.client
+      .call(this.requests.ethCall(txObj))
+      .then(isValidCallRequest)
+      .then(({ result }) => {
```

Comment:
Could use implict return here (and in the function itself) to avoid brace / return keyword noise

Reply:
ack